examples/text-generation/README.md

```diff
@@ -264,6 +264,35 @@ set the following environment variables before running the command: `PT_ENABLE_I
 
 You will also need to add `--torch_compile` in your command.
 
+### Running with Tesor parallel strategy
+#### Attribution
```

Comment:
I think you can remove that line, let's put it in a "box" as suggested below

Reply:
I have added in the box, but not sure if this syntax had to be preserved [!WARNING]